miden-crypto/src/merkle/mmr/full.rs

```diff
@@ -54,19 +54,17 @@ impl Mmr {
 
     /// Constructor for an empty `Mmr`.
     pub fn new() -> Mmr {
-        Mmr { forest: 0, nodes: Vec::new() }
+        Mmr {
+            forest: MountainRange::empty(),
+            nodes: Vec::new(),
+        }
     }
 
     // ACCESSORS
     // ============================================================================================
 
-    /// Returns the MMR forest representation.
-    ///
-    /// The forest value has the following interpretations:
-    /// - its value is the number of elements in the forest
-    /// - bit count corresponds to the number of trees in the forest
-    /// - each true bit position determines the depth of a tree in the forest
-    pub const fn forest(&self) -> usize {
+    /// Returns the MMR representation. See [`MountainRange`].
+    pub const fn mountain_range(&self) -> MountainRange {
```

Comment:
I now there was a discussion about this, but it feels somewhat weird to use `mountain_range()` methods and `MountainRange` type. Technically `Mmr` is the "mountain range" and if we were to spell it out `Mmr` would be `MerkleMountainRange`.

I agree that `Forest` is not ideal either - but at least it is not just an expansion of the `Mmr` abbreviation.

So, not a super strong opinion, but my vote would be to keep this method as `forest()` and maybe even rename `MountainRange` to `Forest` as well.

An alternative could be to find a better name (e.g., maybe `MmrHeader` or something like that), but I'd leave that for a different PR.

Reply:
My vote goes to `Forest` as explained in my top-level comment: https://github.com/0xMiden/crypto/pull/400#issuecomment-2849133841

I made the change back to `Forest` as the last commit in this PR. If others disagree, it's easy to reverse.